subworkflows/local/utils_nfcore_differentialabundance_pipeline/main.nf

```diff
@@ -128,8 +143,62 @@ workflow PIPELINE_COMPLETION {
 //
 // Check and validate pipeline parameters
 //
-def validateInputParameters() {
+def validateInputParameters(paramsets) {
```

Comment:
Thanks for moving all of the checks to this subworkflow :) it makes more sense now